scripts/system/html/js/miniTablet.js

```diff
@@ -130,21 +138,27 @@
 
     function onLoad() {
         muteButton = document.getElementById("mute");
-        muteImage = document.getElementById("mute-img");
         gotoButton = document.getElementById("goto");
-        gotoImage = document.getElementById("goto-img");
         expandButton = document.getElementById("expand");
 
         connectEventBridge();
 
         document.body.addEventListener("mouseenter", onBodyHover, false);
         document.body.addEventListener("mouseleave", onBodyUnhover, false);
 
-        muteButton.addEventListener("mouseenter", onButtonHover, false);
-        gotoButton.addEventListener("mouseenter", onButtonHover, false);
+        if (muteButton) {
+            muteImage = document.getElementById("mute-img");
+            muteButton.addEventListener("mouseenter", onButtonHover, false);
+            muteButton.addEventListener("click", onMuteButtonClick, true);
+        }
+
+        if (gotoButton) {
+            gotoImage = document.getElementById("goto-img");
+            gotoButton.addEventListener("mouseenter", onButtonHover, false);
+            gotoButton.addEventListener("click", onGotoButtonClick, true);
+        }
+
         expandButton.addEventListener("mouseenter", onButtonHover, false);
```

Comment:
should there be similar checks for expandButton?

Reply:
No, not necessary. The script is used in two places now: the original miniTablet.html and the new miniHandsTablet.html. The former has "goto", "mute", and "expand" buttons while the latter only has an "expand" button; hence the need to check "goto" and "mute" but not "expand".